### PR TITLE
refactor: Move `DocsRoute` from admin to separate docs-browser package

### PR DESCRIPTION
**Pull request by rzueger**
The new docs-browser app no is included in the docs route in the
admin app but can also be used as a standalone app (creates a
hash history in this case so that it can be used as a widget in
CMS pages)

Refs: TOCDEV-3043

### CODE REVIEW

File: packages/docs-browser/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "prepublish": "npm run compile:prod"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "tocco-app-extensions": "*",
+    "tocco-test-util": "*",
+    "tocco-theme": "*",
+    "tocco-ui": "*",
+    "tocco-util": "*",
+    "tocco-delete": "*",
```
Comment (MethodenMann) [inline packages/docs-browser/package.json:17]: order alphabetically

File: packages/admin/src/routes/docs/route.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+  }}/>
+)
 
 export default {
-  container: DocsRoute,
-  reducers: {
-    docs: combineReducers({path, create})
-  },
-  sagas: [pathSagas, createSagas]
+  container: DocsBrowserApp
```
Comment (rzueger) [inline packages/admin/src/routes/docs/route.js:14]: make sure the route gets loaded in a separate chunk

File: packages/admin/src/routes/docs/route.js [hunk 3]
```diff
@@ -1,13 +1,15 @@
-import {combineReducers} from 'redux'
+import React from 'react'
```
Comment (MethodenMann) [inline packages/admin/src/routes/docs/route.js:1]: please check if this is loaded correctly (async in own chunk)

File: packages/docs-browser/src/main.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      events,
+      actions: [],
+      publicPath,
+      textResourceModules: ['component', 'common', 'actions', 'entity-list', 'entity-detail', packageName]
+    }
+  )
+}
+
+(() => {
+  if (__DEV__ && __PACKAGE_NAME__ === packageName) {
```
Comment (MethodenMann) [inline packages/docs-browser/src/main.js:75]: I made a crucial change here. Please refer to entity-browser main.js (i will adjust the plop template later)

File: packages/docs-browser/src/dev/textResources.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[
```
Comment (MethodenMann) [inline packages/docs-browser/src/dev/textResources.json:1]: add used docs textresources here? Its kind of a documentation and overview plus will generate nice labels if in mock mode.

**Review by MethodenMann — APPROVED** [review]

**Pull request merged**